src/DataValues/DecimalValue.php

```diff
@@ -321,6 +321,23 @@ public function getFractionalPart() {
 	}
 
 	/**
+	 * Returns a DecimalValue with the same digits as this one, but with any trailing zeros
+	 * after the decimal point removed. If there are no trailing zeros after the decimal
+	 * point, this method will return $this.
+	 *
+	 * @return DecimalValue
+	 */
+	public function trim() {
```

Comment:
This sounds like it modifies the original value, but instead it returns a new one. I think it should say that.